packages/core/src/attestation/Attestation.chain.ts

```diff
@@ -155,3 +156,13 @@ export async function reclaimDeposit(
   )
   return tx
 }
+
+async function queryRawDepositAmount(): Promise<U128> {
```

Comment:
Is there a use-case to `queryRawDepositAmount`, it seems most will use the `queryDepositAmount`?

Reply:
Not really, but I did it to maintain consistency with the other functions as well. The lowest level fetching directly from the blockchain, and a higher level decoding those values as needed.

Reply:
Indeed, the function is not even exported, so not usable outside of this module.